File: crowdRiseWeb-master/crowdRiseWeb-master/src/CrowdRiseBundle/Entity/Financement.php
<?php

namespace CrowdRiseBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Financement
 *
 * @ORM\Table(name="financement", indexes={@ORM\Index(name="id_projet", columns={"id_projet"}), @ORM\Index(name="id_membre_3", columns={"id_membre"}), @ORM\Index(name="moyen", columns={"moyen"})})
 * @ORM\Entity
 */
class Financement
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var float
     *
     * @ORM\Column(name="somme", type="float", precision=10, scale=0, nullable=false)
     */
    private $somme;

    /**
     * @var \Moyen
     *
     * @ORM\ManyToOne(targetEntity="Moyen")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="moyen", referencedColumnName="nom")
     * })
     */
    private $moyen;

    /**
     * @var \Projet
     *
     * @ORM\ManyToOne(targetEntity="Projet")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_projet", referencedColumnName="id")
     * })
     */
    private $idProjet;

    /**
     * @var \Membre
     *
     * @ORM\ManyToOne(targetEntity="Membre")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_membre", referencedColumnName="id")
     * })
     */
    private $idMembre;
    
    function __construct($somme, $idProjet, $idMembre) {
        $this->somme = $somme;
        $this->idProjet = $idProjet;
        $this->idMembre = $idMembre;
        $this->date=new \DateTime();
    }

    
    function getId() {
        return $this->id;
    }

    function getDate() {
        return $this->date;
    }

    function getSomme() {
        return $this->somme;
    }

    function getMoyen() {
        return $this->moyen;
    }

    function getIdProjet() {
        return $this->idProjet;
    }

    function getIdMembre() {
        return $this->idMembre;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setDate(\DateTime $date) {
        $this->date = $date;
    }

    function setSomme($somme) {
        $this->somme = $somme;
    }

    function setMoyen(\Moyen $moyen) {
        $this->moyen = $moyen;
    }

    function setIdProjet(\Projet $idProjet) {
        $this->idProjet = $idProjet;
    }

    function setIdMembre(\Membre $idMembre) {
        $this->idMembre = $idMembre;
    }





}

File: CrowdRiseMobile-master/CrowdRiseMobile-master/src/Entity/Financement.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Entity;



/**
 *
 * @author Aymen
 */
public class Financement {
    int id;
    String moyen;
    String date;
    Membre membre;
    Projet projet;
    double somme;

    public Financement(int id, String moyen, String date, Membre membre, Projet projet, double somme) {
        this.id = id;
        this.moyen = moyen;
        this.date = date;
        this.membre = membre;
        this.projet = projet;
        this.somme = somme;
    }

    public Financement(String moyen, String date, Membre membre, Projet projet, double somme) {
        this.moyen = moyen;
        this.date = date;
        this.membre = membre;
        this.projet = projet;
        this.somme = somme;
    }

    public Financement(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getMoyen() {
        return moyen;
    }

    public void setMoyen(String moyen) {
        this.moyen = moyen;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public Membre getMembre() {
        return membre;
    }

    public void setMembre(Membre membre) {
        this.membre = membre;
    }

    public Projet getProjet() {
        return projet;
    }

    public void setProjet(Projet projet) {
        this.projet = projet;
    }

    public double getSomme() {
        return somme;
    }

    public void setSomme(double somme) {
        this.somme = somme;
    }

    
    public String toString() {
        return "Financement{" + "id=" + id + ", moyen=" + moyen + ", date=" + date + ", membre=" + membre + ", projet=" + projet + ", somme=" + somme + '}';
    }
    
    
    
    
}
